src/test/run-pass/discriminant_value.rs

```diff
@@ -1,5 +1,5 @@
 #![allow(stable_features)]
-#![feature(core, core_intrinsics)]
+#![feature(arbitrary_enum_discriminant, core, core_intrinsics)]
```

Comment:
It would be good to move this file to the folder aforementioned; Moving `discriminant_value-wrapper.rs` would also be good.